agents-api/agents_api/routers/docs/bulk_delete_docs.py

```diff
@@ -0,0 +1,50 @@
+from typing import Annotated
+from uuid import UUID
+
+from fastapi import Depends
+from starlette.status import HTTP_202_ACCEPTED
+
+from ...autogen.openapi_model import (
+    BulkDeleteDocsRequest,
+    ListResponse,
+    ResourceDeletedResponse,
+)
+from ...dependencies.developer_id import get_developer_id
+from ...queries.docs.bulk_delete_docs import bulk_delete_docs as bulk_delete_docs_query
+from .router import router
+
+
+@router.delete(
+    "/agents/{agent_id}/docs:bulk_delete", status_code=HTTP_202_ACCEPTED, tags=["docs"]
+)
+async def bulk_delete_agent_docs(
+    agent_id: UUID,
+    data: BulkDeleteDocsRequest,
+    x_developer_id: Annotated[UUID, Depends(get_developer_id)],
+) -> ListResponse[ResourceDeletedResponse]:
+    deleted_docs = await bulk_delete_docs_query(
+        developer_id=x_developer_id,
+        owner_id=agent_id,
+        owner_type="agent",
+        data=data,
+    )
+
+    return ListResponse[ResourceDeletedResponse](items=deleted_docs)
+
+
+@router.delete(
+    "/users/{user_id}/docs:bulk_delete", status_code=HTTP_202_ACCEPTED, tags=["docs"]
```

Comment:
I believe it's better to use a standard convention for the path: `/users/{user_id}/docs/bulk_delete`

Reply:
cant we do `DELETE /users/{user_id}/docs` ?